MH4U Dex/CoreDataController.m

```diff
@@ -46,88 +48,79 @@ + (CoreDataController *)sharedCDController
 
 #pragma mark - Initial Data Loading
 
+- (void)tryLoadSequence
+{
+    if ([[NSUserDefaults standardUserDefaults] boolForKey:NeedsReloadKey]) {
+        NSLog(@"Resetting Core Data");
+        [self resetCoreData];
```

Comment:
There's also no `DLog` macro in here at the moment—the current Xcode default project template doesn't include a `.pch` file, so it'd have to be added manually in order to put a `DLog` macro in it.